libsol/spl_token_instruction.c

```diff
@@ -105,6 +109,8 @@ static int parse_initialize_multisig_spl_token_instruction(
     BAIL_IF(info->body.m > Token_MAX_SIGNERS);
 
     BAIL_IF(instruction_accounts_iterator_next(&it, &info->multisig_account));
+    // Skip rent sysvar
+    BAIL_IF(instruction_accounts_iterator_next(&it, NULL));
```

Comment:
This line is the bugfix.

The similar changes above in this file are to prevent this type of bug from sneaking in to other instruction parsers in the future.